Desktop Viewer/Classes/LoggerTCPTransport.m

```diff
@@ -408,15 +409,14 @@ - (NSString *)description
 
 - (BOOL)canDoSSL
 {
-	// This method can BLOCK THE CURRENT THREAD and run security dialog UI from the main thread
 	LoggerAppDelegate *appDelegate = (LoggerAppDelegate *)[[NSApplication sharedApplication] delegate];
 	if (!appDelegate.serverCertsLoadAttempted)
 	{
-		dispatch_sync(dispatch_get_main_queue(), ^{
+		dispatch_async(dispatch_get_main_queue(), ^{
 			NSError *error = nil;
 			if (![appDelegate loadEncryptionCertificate:&error])
 			{
-				[NSApp performSelector:@selector(presentError:) withObject:error afterDelay:0];
+				[NSApp presentError:error];
```

Comment:
I think you are looking at b03681031fb0a58349c98dd2b60db66ffb9cb50e which implemented an incorrect way to load the certificates. This has been fixed in a later commit: ee680b9505de2fe71fb073d2f9014462ab51be18.